cypress/integration/html-reader/location-sync.ts

```diff
@@ -16,9 +16,8 @@ it('should remember location between page visits', () => {
     .contains('Chapter 10. A Bosom Friend.')
     .should('be.visible');
 
-  cy.log('Go back home and to another book and then back');
-  cy.visit('/');
-  cy.findByRole('link', { name: 'Moby Dick' }).click();
```

Comment:
I don't think can navigate to a different page with mouse clicks, we have to visit the page directly.

Reply:
I think we can, this seemed to work for me, no?

Reply:
It was working locally at least, and it should work

Reply:
But oh well this is fine too

Reply:
It was throwing [this](https://dashboard.cypress.io/projects/79fvf2/runs/318/test-results/c3741327-e33f-4eec-8faf-5ed67c8cf20f) error when running on Github actin. localhost seems fine.

Reply:
Ah okay yea I _believe_ that is because it wasn't waiting long enough before trying to access the iframe. But I could be wrong. Anyways, this should be fine